Content.Shared/Actions/BaseActionComponent.cs

```diff
@@ -120,6 +120,12 @@ public abstract partial class BaseActionComponent : Component
     /// </summary>
     [ViewVariables] public EntityUid? AttachedEntity;
 
+    /// <summary>
+    ///     The preferred entity to raise an action event on.
+    /// </summary>
```

Comment:
```suggestion
    /// <summary>
    ///     If true, this will cause the the action event to always be raised directed at the action performer/user instead of the action's container/provider.
    /// </summary>
```